src/main/java/cloud/fogbow/rcs/core/intercomponent/xmpp/RemoteMethod.java

```diff
@@ -2,7 +2,8 @@
 
 public enum RemoteMethod {
 
-    REMOTE_GET_SERVICE("remoteGetService");
+    REMOTE_GET_SERVICE("remoteGetService"),
+    REMOTE_GET_SERVICES("remoteGerServices");
```

Comment:
typo: "remoteGetServices". also, i agree with @wassoares : maybe we name this differently as to avoid confusion